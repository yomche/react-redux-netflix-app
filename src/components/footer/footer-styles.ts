import styled from 'styled-components';

export const StyledFooter = styled.footer`
    width: 1200px;
    height: 70px;
    margin: auto;
    background-color: #808080;
    display:flex;
    justify-content:space-around;
    align-items:center;
`;

export const StyledCopyright = styled.p`
    color: #ff0000;
    font-weight: bold;
    font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
`;
