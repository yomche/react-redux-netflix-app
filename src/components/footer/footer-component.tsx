import React from 'react';
import { LogoComponent } from '../logo/logo-component';
import { StyledFooter, StyledCopyright } from './footer-styles';

export const FooterComponent = ():JSX.Element => (
  <StyledFooter>
    <LogoComponent />
    <StyledCopyright>ALL RIGHTS RESERVED, 2021</StyledCopyright>
  </StyledFooter>
);
