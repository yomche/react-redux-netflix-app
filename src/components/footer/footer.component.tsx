import React, { FC } from 'react';
import { Logo } from '../logo/logo.component';
import { StyledFooter, StyledCopyright } from './footer.styles';

export const Footer: FC = () => (
  <StyledFooter>
    <Logo />
    <StyledCopyright>ALL RIGHTS RESERVED, 2021</StyledCopyright>
  </StyledFooter>
);
