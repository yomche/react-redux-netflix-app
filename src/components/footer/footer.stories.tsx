import React, { ComponentProps } from 'react';

import { Story } from '@storybook/react';

import { Footer } from './footer.component';

export default {
  title: 'Components/Footer',
  component: Footer,
};

const Template: Story<ComponentProps<typeof Footer>> = () => <Footer />;

export const Primary = Template.bind({});
Primary.args = {};
