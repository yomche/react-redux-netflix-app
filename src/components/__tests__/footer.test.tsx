import React from 'react';
import { render } from '@testing-library/react';

import { Footer } from '../footer/footer.component';

describe('< Footer /> component', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
