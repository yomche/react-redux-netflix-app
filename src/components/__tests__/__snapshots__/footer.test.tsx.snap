// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< Footer /> component renders correctly 1`] = `
<DocumentFragment>
  <footer
    class="StyledFooter-sc-1nuij0y-0 jppfwh"
  >
    <div
      class="StyledLogo-sc-1wiya5k-0 ehakiy"
    />
    <p
      class="StyledCopyright-sc-1nuij0y-1 gpUgup"
    >
      ALL RIGHTS RESERVED, 2021
    </p>
  </footer>
</DocumentFragment>
`;
